src/audio/copier/copier.c

```diff
@@ -99,31 +99,31 @@ static int create_endpoint_buffer(struct comp_dev *parent_dev,
 	 */
 	if (cd->direction == SOF_IPC_STREAM_PLAYBACK) {
 		if (config->type == SOF_COMP_HOST) {
-			config->frame_fmt = in_frame_fmt;
```

Comment:
config->frame_fmt is redefined here and adjusted according to audio_stream_fmt_conversion

Reply:
@RanderWang ok, I think this is wrong. This `config` isn't `const`. I'll drop this patch. Thanks for catching!